hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSink.java

```diff
@@ -274,8 +276,10 @@ public void replicateEntries(List<WALEntry> entries, final CellScanner cells,
       this.metrics.setAgeOfLastAppliedOp(entries.get(size - 1).getKey().getWriteTime());
       this.metrics.applyBatch(size + hfilesReplicated, hfilesReplicated);
       this.totalReplicatedEdits.addAndGet(totalReplicated);
+      // TODO(baugenreich) -> does it make sense to reset the counter here ??
     } catch (IOException ex) {
```

Comment:
actually, seeing how this just re-throws anyway, i'd probably just change this catch to Exception instead of IOException.